res/database/Default/database_news.xml

```diff
@@ -2,6 +2,2994 @@
 <tvtdb>
 	<version value="3" />
 	<allnews>
+
+		<!-- 'Real news reel' - Historical news sorted by date (some fictional 'alternative universe' variations may occur) -->
+
+			<!-- 1972-general -->
+
+		<news id="b5140fd5-e3af-40f4-a985-6d5b0a742271" type="0" creator="8936" created_by="jorgaeff" comment="https://en.wikipedia.org/wiki/Space_Shuttle_program | previously part of thread 5328557c-be27-46d2-acc0-a8ae2901eff3 | previously wrong date 1985">
+			<title>
+				<de>NASE startet Space-Shuttle-Programm</de>
+				<en>NASE launches space shuttle program</en>
+				<pl>NASE uruchamia program promów kosmicznych</pl>
+			</title>
+			<description>
+				<de>Gestern der Mond und morgen der Mars? Die nordamerikanische Raumfahrtbehörde NASE möchte auch im All die Nase vorn haben und startet ein neues Raumfahrtprogramm. Es soll der Erkundung neuer Planeten dienen.</de>
+				<en>The moon yesterday and Mars tomorrow? The North American space agency NASE wants to be in the lead also in space and launches a new space program. It is to help the exploration of new planets.</en>
+				<pl>Wczoraj Księżyc jutro Mars? Północnoamerykańska agencja kosmiczna NASE chce być liderem również w kosmosie i uruchamia nowy program kosmiczny. Ma on pomóc w eksploracji nowych planet.</pl>
+			</description>
+			<!-- flags: 2 = unique event, not auto-reuseable -->
+			<data genre="3" price="0.6" quality="60" fictional="0" flags="2" happen_time="4,1980,6,1" /> <!-- TODO: actually in 1972; 'adjust' once playing in the 1970s is possible -->
```

Comment:
you mean because the news is in 1981? Well you can start to play in 1980. Players don't see the ones of the later years if they don't play that long. Eventually there should / will be more historical entries for every year.